marker_driver_rustc/src/context.rs

```diff
@@ -113,4 +113,8 @@ impl<'ast, 'tcx: 'ast> DriverContext<'ast> for RustcContext<'ast, 'tcx> {
         let api_str: &'ast str = unsafe { std::mem::transmute(rustc_str) };
         api_str
     }
+
+    fn resolve_method_target(&'ast self, _id: ExprId) -> ItemId {
+        todo!()
+    }
```

Comment:
I know how to implement this, but I want to do some slight adjustments beforehand, which will result in superficial changes. For the review, I believe better to split the implementation in two PRs.